packages/compass-e2e-tests/tests/auto-connect.test.ts

```diff
@@ -91,16 +84,17 @@ describe('Automatically connecting from the command line', function () {
 
   async function waitForConnectionSuccessAndCheckConnection(
     compass: Compass,
-    expectedTitle = connectionStringSuccessTitle
+    connectionName: string
   ) {
-    const connectionName = connectionNameFromString(connectionStringSuccess);
     await compass.browser.waitForConnectionResult(connectionName, {
       connectionStatus: 'success',
     });
-    const sidebarTitle = await compass.browser
-      .$(Selectors.SidebarTitle)
-      .getText();
-    expect(sidebarTitle).to.eq(expectedTitle);
+    if (!TEST_MULTIPLE_CONNECTIONS) {
```

Comment:
No else necessary because in MC world waitForConnectionResult already checks that the exact connectionName was connected and there is no sidebar title that changes accordingly.